packages/esm-patient-medications-app/src/config-schema.ts

```diff
@@ -30,6 +30,12 @@ export const configSchema = {
     _default: 99,
     _description: 'The maximum number of days for medication dispensing.',
   },
+  debounceDelayInMs: {
+    _type: 'integer',
```

Comment:
[_type](https://github.com/openmrs/openmrs-esm-core/blob/ff09ef39a8477ca2b7ddeb9c0d94ca5efc076d50/packages/framework/esm-config/src/types.ts#L1) must be one of the properties in the `Type` enum. In this case, it should be:

```suggestion
    _type: Type.Number,
```